extras/configure_gluster_collectd.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Install and configure gluster collectd package
+  hosts: all
+  vars:
+    copr_url: "https://copr-be.cloud.fedoraproject.org/results/redara"
+    repo: "gluster-collectd/fedora-27-x86_64/00774074-gluster-collectd"
+    package: "gluster-collectd-1.0.0-1.fc27.noarch.rpm"
```

Comment:
can we have the package as gluster-collectd-1.0.0.1 without the distro and arch specified?

Reply:
When providing the url we cannot this. We have to give the complete file name.

Reply:
As Sahina said. I believe this should be packaged as gluster-collectd-1.0.0.1 rpm upstream and downstream. @sabose @sac